compiler/codegen.js

```diff
@@ -2183,7 +2234,90 @@ const generateCall = (scope, decl, _global, _name, unusedValue = false) => {
       let args = decl.arguments;
       let locals = [];
 
-      if (indirectMode === 'vararg') {
+      if (indirectMode === 'array') {
```

Comment:
Forgot to remove this too? :sweat_smile:

Reply:
Woops :sweat_smile: